src/System.Management/src/System/Management/WMIGenerator.cs

```diff
@@ -4861,7 +4861,12 @@ bool InitializeCodeGenerator(CodeLanguage lang)
                         strProvider = "Visual Basic.";
                         cp = new VBCodeProvider();
                         break;
-            
+
+                    case CodeLanguage.JScript:
+                        strProvider = "JScript.NET.";
+                        bSucceeded = false; // JScriptCodeProvider does not exist on CoreFx
+                        break;
```

Comment:
There a bunch of code for other `bSucceeded = false;` cases for other languages below, e.g. VJSharp, Mcpp.  What's the reason they need all that assembly-related code?  If it's not necessary, can/should it be removed?  If it is necessary, is similar code necessary here?

Reply:
So far we have adopted the policy of keeping this code as close as possible to the desktop code, that's why those are left there. Notice that `System.Management` is no longer being developed (https://msdn.microsoft.com/en-us/library/aa393964(v=vs.85).aspx), this work is now done under a different namespace.

The assemblies for Mcpp and VJSharp are not shipped with desktop but in principle an app can ship them and try to generate the code, that said I didn't try it myself (I still can see someone using Mcpp, but VJSharp just recently got out of support window: 
> Support for Visual J# 2.0 Redistributable Package Second Edition ends on 10/10/2017. For more information, see https://msdn.microsoft.com/en-us/vstudio/bb188593.

Reply:
Ok.  Do we need a test for this change?

Reply:
I added the test, it indeed needed the GC snippet above to not leave files on the disk. Notice that currently we don't run these tests against netfx on CI but I do it manually whenever I change anything.